CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 * [CHANGE]
 * [FEATURE]
 * [ENHANCEMENT]
-* [BUGFIX]
+* [BUGFIX] Linux CPU: Cache CPU metrics
```

Comment:
missing # numer

Reply:
I prefer to keep the changelog entries short. But I can expand it a bit.

Reply:
But then:

[BUGFIX] Linux CPU: Workaround buggy counter resets